javascript-local-storage/data.js

```diff
@@ -0,0 +1,9 @@
+/* exported todos */
+
+const previousTodosJSON = localStorage.getItem('javascript-local-storage');
+let todos = JSON.parse(previousTodosJSON);
```

Comment:
You should have an `if` statement here, checking if `previousTodosJson` is not null